Server/config/passport.config.js

```diff
@@ -1,13 +1,49 @@
+const passport = require('passport');
+var LocalStrategy = require('passport-local').Strategy;
+const passportJWT = require('passport-jwt');
+const JWTStrategy = passportJWT.Strategy;
+const ExtractJwt = require('passport-jwt').ExtractJwt;
+const bcrypt = require('bcrypt');
+const userModel = require('../models/user.model');
+const saltRounds = 10;
 
-/*
- * @class passportConfig
- */
-class passportConfig {
-
-    /**
-    * Initializes the passport config
-    */
+// Local passport strategy is used to authenticating user and generating a token  
+passport.use(new LocalStrategy({
```

Comment:
Please split it into 2 functions for readability, e.g.
`// Strategy to authenticate a user
const localStrategy = new LocalStrategy({...............})`
// Utilize the strategy
passport.use(localStrategy);`

Reply:
Updated as suggested